src/core/choroplethFeature.js

```diff
@@ -0,0 +1,245 @@
+  /**
+   * Get/Set choropleth accessor
+   *
+   * @returns {geo.feature.choropleth}
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  this.choropleth = function (arg1, arg2) {
+    if (arg1 === undefined) {
+      return m_choropleth;
+    }
+    if (typeof arg1 === 'string' && arg2 === undefined) {
+      return m_choropleth[arg1];
+    }
+    if (arg2 === undefined) {
+      var choropleth = $.extend(
+        {},
+        m_choropleth,
+        arg1
+      );
+      m_choropleth = choropleth;
+    } else {
+      m_choropleth[arg1] = arg2; //if you pass in accessor for prop
+    }
+    m_this.modified();
+    return m_this;
+  };
+
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * A uniform getter that always returns a function even for constant values.
+   * If undefined input, return all the choropleth values as an object.
+   *
+   * @param {string|undefined} key
+   * @return {function}
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  this.choropleth.get = function (key) {
+    if (key === undefined) {
+      var all = {}, k;
+      for (k in m_choropleth) {
+        if (m_choropleth.hasOwnProperty(k)) {
+          all[k] = m_this.choropleth.get(k);
+        }
+      }
+      return all;
+    }
+    return geo.util.ensureFunction(m_choropleth[key]);
+  };
+
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * A method that adds a polygon feature to the current layer.
+   *
+   * @param {array} coordinateArray
+   * @param {geo.color} fillColor
+   * @return {geo.feature}
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  this._addPolygonFeature = function(coordinateArray, fillColor){
+    return m_this.layer()
+      .createFeature('polygon', {
+      })
+      .data([{
+        type: 'Polygon',
+        coordinates: coordinateArray
+      }])
+      .polygon(function (d) {
+        return {
+          'outer': d.coordinates[0],
+          'inner': d.coordinates[1] // undefined but ok
+        };                              
+      })
+      .position(function(d){
+        return {
+          x: d[0],
+          y: d[1]
+        };
+      })
+      .style({
+        'fillColor': fillColor
+      });
+  };
+
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * A method that adds polygons from a given feature to the current layer.
+   *
+   * @param {} geoJsonFeature
+   * @param geo.color
+   * @return [{geo.feature}] 
+   */
+  ////////////////////////////////////////////////////////////////////////////          
+  this._featureToPolygons = function (feature, fillValue) {
+
+    var arrayOfPolygonFeatures = [];
+    
+    if (feature.geometry.type === 'Polygon'){
+      arrayOfPolygonFeatures = [
+        m_this
+          ._addPolygonFeature(feature.geometry.coordinates, fillValue)
```

Comment:
Because our interface is more general than geojson, this should be wrapped in an accessor that returns the array of coordinates like the `value` accessor.